std-bits/base/src/main/java/org/enso/base/enso_cloud/telemetry/TelemetryLogMessage.java

```diff
@@ -0,0 +1,44 @@
+package org.enso.base.enso_cloud.telemetry;
+
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.fasterxml.jackson.databind.node.TextNode;
+import java.util.Objects;
+import org.enso.base.enso_cloud.logging.LogMessage;
+
+public final class TelemetryLogMessage extends LogMessage {
```

Comment:
```suggestion
final class TelemetryLogMessage extends LogMessage {
```
only accessed from the same package. No reason for being `public` I guess.

Reply:
Done in c0e771c1273cc8ae6e7818805f5c296bdd40d82b